src/sections/TransactionInfoSection.tsx

```diff
@@ -115,22 +146,29 @@ const TransactionInfoSection = () => {
                         content={
                           <>
                             <LoadingSpinner style={{ marginRight: 5 }} size={15} />
-                            <span>Unconfirmed</span>
+                            <span>Pending</span>
                           </>
                         }
                       />
                     </td>
                   )}
                 </TableRow>
-                {isTxConfirmed(txInfo) && txInfo.blockHash && (
+                {isTxConfirmed(txInfo) && txInfo.blockHash && txBlock && (
                   <TableRow>
-                    <td>Block Hash</td>
+                    <td>Block</td>
                     <td>
-                      <TightLink
-                        to={`../blocks/${txInfo.blockHash || ''}`}
-                        text={txInfo.blockHash || ''}
-                        maxWidth="550px"
-                      />
+                      <SimpleLink to={`../blocks/${txInfo.blockHash || ''}`}>{txBlock?.height.toString()}</SimpleLink>
```

Comment:
I see that you changed the value to be the block height instead of the block hash. Then maybe we should also update the label of this entry from `<td>Block</td>` to `<td>Block Height</td>` to be more accurate?

Reply:
I followed Etherscan's approach here, where the link redirects the user to the block page and the height serves as an ID. 
But as we have multiple chain, it's true that the height can't be considered as a unique identifier. To be perfectly correct, we should specify the Chain Index in the TX details as well. WDYT?

Reply:
Ah, ok, didn't check Etherscan, I only noticed the change in the value but not a change in the label of the table row.

I don't know how important is to mention in which chain the block of the transaction exists, in the transaction page. It is already mentioned in the block page. As long as we link to the block page from the transaction page, it seems enough.

Unless of course you want to use block height AND chain id for unique id. Cause block height alone is not unique. Block hash is however.

Reply:
Another idea:

<img width="1222" alt="image" src="https://user-images.githubusercontent.com/1579899/166915757-716beda9-fbcb-48ad-8da2-c8c5b51e61ec.png">

Reply:
What about 

![image](https://user-images.githubusercontent.com/6715943/166917772-42ae094c-1be6-4abd-9ce6-22af4ad75625.png)

Reply:
(Done)

Reply:
Could be nice to come up with a better format to display the chain with the block hash in the future. Maybe the chain can be a subscript or superscript or something... having to hover for it doesn't feel great, especially since a block hash is a technical value and so knowing the chain is important.

Reply:
All the block details are one click away. Let's see if users complain :)